sys/config/include/config/config.h

```diff
@@ -157,14 +162,40 @@ struct conf_handler {
      * The name of the conifguration item/subtree
      */
     char *ch_name;
+
+    /**
+     * Whether to use the extended callbacks.
+     * 0=standard
+     * 1=extended
+     */
+    uint8_t ch_ext : 1;
```

Comment:
I think this should rather be `bool`, although it would not be a problem to use a bitfield in practice, unless someone wrote a configuration to flash and moved that flash to a CPU with different endiannes! :-P

Reply:
I agree, it should be a bool.  I think I was bothered by the size increase of this struct so I went overboard.  But since there is only one field like this, we're not saving any space anyway.  I'll change it to bool.